fix: correct grammatical error (#134)
smartmind-team/thanosql-docs: docs/en/getting_started/data_upload.md

---

> title: How to upload your data to the ThanoSQL Workspace

Reviewer: ```suggestion
title: How to Upload Your Data to the ThanoSQL Workspace
```

---

> # __How to upload your data to the ThanoSQL Workspace__ 

Reviewer: ```suggestion
# __How to Upload Your Data to the ThanoSQL Workspace__ 
```

## __Preface__

> ThanoSQL Workspace's users can upload data using two ways as shown below.

Reviewer: ```suggestion
The ThanoSQL workspace's users can upload data in two ways, as shown below.
```

!!! note "To upload data"
    1. Uploading your local data into your workspace environment
    2. Uploading local data by SFTP server connection using a private key

> ## __1. Uploading your local data in a workspace environment__

Reviewer: ```suggestion
## __1. Uploading your local data into a workspace environment__
```

> To upload local data to ThanoSQL Workspace, follow these steps:

Reviewer: ```suggestion
Follow these steps to upload local data to the ThanoSQL workspace:
```

---
title: How to upload your data to the ThanoSQL workspace
---

> # __How to upload your data to the ThanoSQL workspace__ 

Reviewer: ```suggestion
# __How to Upload your Data to the ThanoSQL Workspace 
```
다른 file들 title보니깐 다 uppercase되어 있는거 같아서 통일 하면 좋을거 같아요. 

`rest_api_thanosql_insert.md`
- How to Upload to ThanoSQL DB -> How to Upload to the ThanoSQL Database

`rest_api_thanosql_query.md`
- How to Query Using ThanoSQL -> How to Query Using ThanoSQL 

다른 file들도 고쳐야 하는지 확인 필요 할거 같아요
